code/modules/maps/planet_types/grass.dm

```diff
@@ -48,6 +48,11 @@
 	if(L && L.client && (L.client.prefs.sfx_toggles & ASFX_AMBIENCE) && !L.ear_deaf)
 		L.playsound_to(get_turf(L),sound('sound/ambience/jungle.ogg', repeat = 1, wait = 0, volume = 25, channel = 1))
 
+/area/exoplanet/grass/Exited(var/mob/living/L, atom/newarea)
```

Comment:
What is this needed for?

Reply:
That stops the ambient jungle sound from playing if the player exits the area. Currently grove planet ambience is bugged and never stops playing. The reason I went about fixing it with this PR was cause I had the same problem with my away site, so I figured I'd might as well.